tools/stimulus/types-RC.yaml

```diff
@@ -706,18 +710,26 @@ EDGE_ATTRIBUTE_COMBINATION:
 HRG:
     CALL: '&%C%'
     INCONV:
-        IN: R_SEXP_to_hrg(%I%, &%C%);
+        IN: |-
+            if (0 != R_SEXP_to_hrg_copy(%I%, &%C%)) {
+              igraph_error("internal error in R_SEXP_to_hrg_copy()", __FILE__, __LINE__, IGRAPH_ENOMEM);
+            }
+            IGRAPH_FINALLY(igraph_hrg_destroy, &%C%);
         INOUT: |-
             if (0 != R_SEXP_to_hrg_copy(%I%, &%C%)) {
-              igraph_error("", __FILE__, __LINE__, IGRAPH_ENOMEM);
+              igraph_error("internal error in R_SEXP_to_hrg_copy()", __FILE__, __LINE__, IGRAPH_ENOMEM);
```

Comment:
this one needs a fix too and then regenerate

Reply:
@Antonov548: Can you please tweak according to Szabolcs's comments?